templates/S3/synapse-external-bucket.j2

```diff
@@ -151,25 +152,8 @@ Resources:
                 - "s3:PutObject"
                 - "s3:PutObjectAcl"
                 - "s3:DeleteObject"
+                - "s3:*MultipartUpload*"
```

Comment:
Clients can already upload files with the the existing permissions. Why add `*MultipartUpload*`?


If you are combining the two statements, then perhaps rename `InternalPutObjectAccess` to simply be`PutObjectAccess`.

Reply:
> Clients can already upload files with the the existing permissions. Why add *MultipartUpload*? 

Are you sure about this?  My understanding is PUT is a single operation upload while [multipart](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html) is a special upload designed for large files.  There are specific [AWS CLI commands for multipart uploading of files](https://aws.amazon.com/premiumsupport/knowledge-center/s3-multipart-upload-cli/).

Reply:
@brucehoff Maybe i'm missing something but I don't see how that document clarifies whether we need `s3:*MultipartUpload*` permission here.  Maybe you can highlight where you feel it's giving us a clue?

Reply:
@zaro0508 Please read this section: https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html#mpuAndPermissions
which says that `s3:PutObject` lets you Create Multipart Upload, Initiate Multipart Upload, Upload Part, Complete Multipart Upload, ...

Reply:
OK,  thanks.  However looking at this again regardless of whether `s3:PutObject` allows uploading files we still want to give `s3:*MultipartUpload*` permission to users so they can list and abort their uploads.